core/domain/role_services.py

```diff
@@ -34,28 +34,20 @@
 ACTION_ACCESS_MODERATOR_PAGE = 'ACCESS_MODERATOR_PAGE'
 ACTION_CREATE_COLLECTION = 'CREATE_COLLECTION'
 ACTION_CREATE_EXPLORATION = 'CREATE_EXPLORATION'
-ACTION_DELETE_ANY_PUBLIC_EXPLORATION = 'DELETE_ANY_PUBLIC_EXPLORATION'
-ACTION_DELETE_OWNED_PRIVATE_EXPLORATION = 'DELETE_OWNED_PRIVATE_EXPLORATION'
-ACTION_EDIT_ANY_COLLECTION = 'EDIT_ANY_COLLECTION'
-ACTION_EDIT_ANY_EXPLORATION = 'EDIT_ANY_EXPLORATION'
-ACTION_EDIT_ANY_PUBLIC_COLLECTION = 'EDIT_ANY_PUBLIC_COLLECTION'
-ACTION_EDIT_ANY_PUBLIC_EXPLORATION = 'EDIT_ANY_PUBLIC_EXPLORATION'
-ACTION_EDIT_OWNED_COLLECTION = 'EDIT_OWNED_COLLECTION'
-ACTION_EDIT_OWNED_EXPLORATION = 'EDIT_OWNED_EXPLORATION'
+ACTION_DELETE_ANY_PUBLIC_ACTIVITY = 'DELETE_ANY_PUBLIC_ACTIVITY'
+ACTION_DELETE_OWNED_PRIVATE_ACTIVITY = 'DELETE_OWNED_PRIVATE_ACTIVITY'
+ACTION_EDIT_ANY_ACTIVITY = 'EDIT_ANY_ACTIVITY'
+ACTION_EDIT_ANY_PUBLIC_ACTIVITY = 'EDIT_ANY_PUBLIC_ACTIVITY'
+ACTION_EDIT_OWNED_ACTIVITY = 'EDIT_OWNED_ACTIVITY'
 ACTION_FLAG_EXPLORATION = 'FLAG_EXPLORATION'
 ACTION_MANAGE_EMAIL_DASHBOARD = 'MANAGE_EMAIL_DASHBOARD'
 ACTION_MANAGE_PROFILE = 'MANAGE_PROFILE'
-ACTION_MODIFY_ROLES_FOR_ANY_EXPLORATION = 'MODIFY_ROLES_FOR_ANY_EXPLORATION'
-ACTION_MODIFY_ROLES_FOR_OWNED_EXPLORATION = (
-    'MODIFY_ROLES_FOR_OWNED_EXPLORATION')
-ACTION_PLAY_ANY_PRIVATE_COLLECTION = 'PLAY_ANY_PRIVATE_COLLECTION'
-ACTION_PLAY_ANY_PUBLIC_COLLECTION = 'PLAY_ANY_PUBLIC_COLLECTION'
-ACTION_PLAY_ANY_PRIVATE_EXPLORATION = 'PLAY_ANY_PRIVATE_EXPLORATION'
-ACTION_PLAY_ANY_PUBLIC_EXPLORATION = 'PLAY_ANY_PUBLIC_EXPLORATION'
-ACTION_PUBLISH_ANY_COLLECTION = 'PUBLISH_ANY_COLLECTION'
-ACTION_PUBLISH_ANY_EXPLORATION = 'PUBLISH_ANY_EXPLORATION'
-ACTION_PUBLISH_OWNED_COLLECTION = 'PUBLISH_OWNED_COLLECTION'
-ACTION_PUBLISH_OWNED_EXPLORATION = 'PUBLISH_OWNED_EXPLORATION'
+ACTION_MODIFY_ROLES_FOR_ANY_ACTIVITY = 'MODIFY_ROLES_FOR_ANY_ACTIVITY'
+ACTION_MODIFY_ROLES_FOR_OWNED_ACTIVITY = 'MODIFY_ROLES_FOR_OWNED_ACTIVITY'
+ACTION_PLAY_ANY_PRIVATE_ACTIVITY = 'PLAY_ANY_PRIVATE_ACTIVITY'
+ACTION_PLAY_ANY_PUBLIC_ACTIVITY = 'PLAY_ANY_PUBLIC_ACTIVITY'
+ACTION_PUBLISH_ANY_ACTIVITY = 'PUBLISH_ANY_ACTIVITY'
+ACTION_PUBLISH_OWNED_ACTIVITY = 'PUBLISH_OWNED_ACTIVITY'
 ACTION_PUBLICIZE_EXPLORATION = 'PUBLICIZE_EXPLORATION'
```

Comment:
Can we call this ACTION_PUBLICIZE_ANY_EXPLORATION? Ditto for UNPUBLICIZE etc. and all other actions that are exploration/collection/activity-related.

I think this is important so that we have a clear distinction between "perform activity on any/owned private/public/{{empty string}} exploration" -- every action name of this form should have these qualifiers.